src/app.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import Header from './header';
+
+import JudgingRoundDisplay from './judging_round_display';
+import JudgingRoundSelector from './judging_round_selector';
+
+
+class App extends React.Component {
+    constructor(props) {
+	super(props);
+	this.state = {
+            judging_round: null,
+	};
+    }
+
+    select_judging_round = (_, data) => {
+	this.setState({judging_round: data.value})
+    }
+
+    render() {
+	return (<div>
```

Comment:
Not a blocker, but I'm still not a fan of this JSX stuff.

Reply:
Won't fix.